packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -146,13 +146,15 @@ const getStyles = (theme: GrafanaThemeV2) => {
         right: 0px;
         bottom: ${theme.spacing(0.25)};
         top: ${theme.spacing(0.25)};
+        z-index: 1;
```

Comment:
nit: I think we have z-indexes in the theme, maybe it needs a new addition?

Reply:
I just realized the only reason we need this is because the datalink suggestions has a wrapper with z-index: 1. Not sure that's needed. Can possibly remove that and this.

Reply:
Tested removing the z-index, works on all places the the DataLinkSuggestions is used in both chrome and ff